File: BasicMath/Amstrong.py
n = int(input("Enter the number: "))

ams = 0
orginal = n
while n > 0:
    rem = n % 10
    ams = ams + (rem * rem * rem)
    n //= 10
if orginal == ams:
    print(f"{ams} is a amstrong number")
else:
    print(f"{ams} is not equal to {orginal} so its not a Amstrong number")

File: BasicMath js/amstrong.js
// 153

// 1 cube + 5 cube + 3 cube = 153

function amstrong(n) {
  let original = n;
  let ams = 0;
  while (n > 0) {
    let rem = n % 10;
    ams = ams + rem * rem * rem;
    n = Math.floor(n / 10);
  }

  if (ams === original) {
    return "Ams";
  } else return "not ams";
}

console.log(amstrong(121));
